# VN100 imu pp data to publish
Header  header
float32 mag_x
float32 mag_y
float32 mag_z
float32 mag_abs
